quality.gradle

```diff
@@ -78,11 +78,13 @@ task pmd(type: Pmd, group: 'verification') {
     }
 }
 
-android {
-    lintOptions {
-        abortOnError true
-        xmlReport true
-        htmlReport true
-        lintConfig file("${project.rootDir}/AndroidCodeQualityConfig/quality/lint/lint.xml")
+afterEvaluate {
```

Comment:
So the problem happens when you apply quality.gradle at root project?

If so, I need update readme to add this note.